src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs

```diff
@@ -447,7 +447,16 @@ internal PSModuleInfo LoadUsingMultiVersionModuleBase(string moduleBase, Manifes
         /// <summary>
```

Comment:
@vexx32, your last commit had 23 failures in `PowerShell-CI-macos`
(These are 5 of the failures)

float/double precision when converting to string.[double]-to-[string] conversion in PowerShell should use the precision specifier G15
```powershell
Expected strings to be the same, but they were different.
Expected length: 3
Actual length:   18
Strings differ at index 3.
Expected: '3.3'
But was:  '3.3000000000000003'
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Language/Parser/Conversions.Tests.ps1: line 540
540:         $value -as [string] | Should -BeExactly $StringConversionResult
```
float/double precision when converting to string.[double]-to-[string] conversion in PowerShell should use the precision specifier G15
```powershell
Expected strings to be the same, but they were different.
Expected length: 3
Actual length:   18
Strings differ at index 3.
Expected: '6.6'
But was:  '6.6000000000000005'
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Language/Parser/Conversions.Tests.ps1: line 540
540:         $value -as [string] | Should -BeExactly $StringConversionResult
```
float/double precision when converting to string.[double]-to-[string] conversion in PowerShell should use the precision specifier G15
```powershell
Expected strings to be the same, but they were different.
Expected length: 16
Actual length:   17
Strings differ at index 15.
Expected: '2.71828182845905'
But was:  '2.718281828459045'
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Language/Parser/Conversions.Tests.ps1: line 540
540:         $value -as [string] | Should -BeExactly $StringConversionResult
```
float/double precision when converting to string.[double]-to-[string] conversion in PowerShell should use the precision specifier G15
```powershell
Expected strings to be the same, but they were different.
Expected length: 16
Actual length:   17
Strings differ at index 16.
Expected: '3.14159265358979'
But was:  '3.141592653589793'
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Language/Parser/Conversions.Tests.ps1: line 540
540:         $value -as [string] | Should -BeExactly $StringConversionResult
```
float/double precision when converting to string.[float]-to-[string] conversion in PowerShell should use the precision specifier G7
```powershell
Expected strings to be the same, but they were different.
Expected length: 3
Actual length:   9
Strings differ at index 3.
Expected: '3.3'
But was:  '3.3000002'
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Language/Parser/Conversions.Tests.ps1: line 540
540:         $value -as [string] | Should -BeExactly $StringConversionResult
```